### ISSUE TITLE
Domain activation is Failed - due to a Certificate Authority Authentication error.

### ISSUE THREAD

**Issue by HMilbradt** [aws-amplify/amplify-console#194]
**App Id**: d3bsiju3l3vwuf
**Region**: us-east-2
**Step**: Domain activation
**Status**: Failed

>**Note**: Do not include information that is sensitive in nature such as your domain name, company etc.

**Issue/question**
I receive this error while attempting to validate my custom domain in the console.  I have set up the CNAME record that is shown in the format ```_1234.example CNAME _1234.example.acm-validations.aws```.  This CNAME record is in Cloudflare, with proxy turned off (DNS only).

I have also tried adding the CAA records found in the link in the error message, as well as deleting the custom domain, deleting the DNS records and re-adding them, and trying a completely different domain.  I have also waited about 12 hours to see if it was a DNS propagation issue, but it doesn't seem likely.

I've also noticed it stays in the "SSL configuration" stage until I add the CNAME record above, after which it starts throwing this error.

Any ideas what can fix this?  Ive noticed in the consoles (tried safari and Firefox) I get the same error message as https://github.com/aws-amplify/amplify-console/issues/187, but I can see that the network tab is polling Amplify.

Thanks

**Error message**

> Error message: The status of this certificate request is "Failed". One or more domain names have failed validation due to a Certificate Authority Authentication error. Learn more: https://docs.aws.amazon.com/acm/latest/userguide/troubleshooting-caa.html

**Additional information**
I have another Amplify project that was configured the same way, using Cloudflare without the proxy.  This other project works fine, and the custom domain is working

**Comment by cslogan-red** [comment]
Hi there, I would go ahead and delete the custom domain and start from scratch and then add it back to the Console. The CAA records are optional, but if you already have them in place in your DNS provider they must be configured per the link you provided, they are not required to connect your domain to Amplify Console. You can verify the records have met the minimum requirements outlined in that documentation for your domain using whatsmydns.net and checking your CAA records once they are configured.

Additionally, when initially configuring the CNAME verification record, most DNS providers automatically add the ".mycustomdomain.com" suffix to the end, so you usually only need to add the part starting with the underscore WITHOUT the domain suffix, for example: "_12a3b456cd", in your host name column. Sometimes manually including the domain suffix can throw off the validation, depending on the provider, thanks!

**Comment by HMilbradt** [comment]
Thanks for the reply, will post an update when I have something working

**Comment by HMilbradt** [comment]
So I was able to get this working with Cloudflare.  I should have clarified in the original post I was adding a subdomain instead of a root domain.

The steps I took to resolve this:

I deleted my CAA records that I had added, deleted the custom domain from Amplify, and removed any other existing CNAME records I had added from before.

The CNAME record I added was `_1234.subdomain CNAME _1234.1234.acm-validations.aws`.  
I also added the CNAME record to the domain `subdomain.example.com CNAME cloudfront.com`.  In Cloudflare, I turned off the proxy for both of these records.  

After waiting a few hours for all the records to propagate, the domain was verified.  At this point I could turn on Cloudflares proxy for the cloudfront domain and everything worked fine.
